packages/dev/mcp/react-aria/src/index.ts

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+/// <reference types="node" />
+import {errorToString} from '../../src/common/utils.js';
+import {startServer} from '../../src/common/server.js';
+
+// CLI entry for React Aria
+(async () => {
+  try {
+    const arg = (process.argv[2] || '').trim();
+    if (arg === '--help' || arg === '-h' || arg === 'help') {
```

Comment:
https://nodejs.org/api/util.html#utilparseargsconfig

Reply:
Can improve these in a follow-up, just wanted to include the package refactor in this PR.